featureflags.go

```diff
@@ -831,27 +822,16 @@ func containsVariant(variantList []FlagVariant, key string) bool {
 }
 
 // extracted as a regular func for testing purposes
-func checkIfSimpleFlagEnabled(key string, distinctId string, rolloutPercentage uint8) (bool, error) {
-	val, err := _hash(key, distinctId, "")
-	if err != nil {
-		return false, err
-	}
-
-	return val <= float64(rolloutPercentage)/100, nil
+func checkIfSimpleFlagEnabled(key, distinctId string, rolloutPercentage uint8) bool {
+	hash := calculateHash(key, distinctId, "")
```

Comment:
Per the below comment, I believe this should be:

```suggestion
	hash := calculateHash(key + ".", distinctId, "")
```

If you make the suggested change below.